.ruby-version

```diff
@@ -1 +1 @@
-2.5.3
\ No newline at end of file
+2.5.3
```

Comment:
Although I'm absolutely all for ensuring that **all** text files are well-terminated (which means ending with a `\n` character), this file is special, as it is only to contain a string, instead of "text". That said, I don't think there is a problem with adding the termination, but every time we change the version via a manager of rubies (e.g. RVM or rbenv) it'll be re-written to a non-terminated line. Therefore, I propose that we don't commit this type of change to it.